engine/config/locales/cy.yml

```diff
@@ -1,20 +1,23 @@
+---
```

Comment:
What does `---` do?

Reply:
Hold on, I can see this is a result of running `normalize keys`

Reply:
Per the yml spec most files "should" start with this sorta thing. It's also auto-generated by a lot of CLI stuff. It's kinda annoying though